packages/l/lact/package.yml

```diff
@@ -14,12 +14,13 @@ description: |
 builddeps  :
     - pkgconfig(gtk4)
     - pkgconfig(libdrm)
-    - blueprint-compiler
     - rust
+    - vulkan-tools
+rundeps    :
+    - vulkan-tools
 setup      : |
     %patch -p1 -i $pkgfiles/0001-Solus-Start-after-systemd-udev.patch
-    %patch -p1 -i $pkgfiles/0001-Solus-Use-run-instead-of-var-run.patch
```

Comment:
What's the reason for dropping this patch?

Reply:
Upstream now uses the /run instead of /var/run

Reply:
Neat.